cogs5e/models/automation/effects/save.py

```diff
@@ -96,13 +97,20 @@ def run(self, autoctx):
                     autoctx.add_pm(str(autoctx.ctx.author.id), out)
                     autoctx.queue(f"**{save_blurb}**: 1d20...{success_str}")
         else:
-            autoctx.meta_queue('{} Save'.format(save_skill[:3].upper()))
+            autoctx.meta_queue('{} Save'.format(stat.upper()))
             is_success = False
 
+        # Disable critical damage state for children #1556
+        old_no_crit = autoctx.args.last('nocrit', default=False, type_=bool)
+        autoctx.args.update({'nocrit': True})
```

Comment:
Why update the arguments rather than the internal crit state (ala `Attack.on_crit()`)? If, for example, some automation existed as such:
```yaml
- type: save
  save: ...
  fail:
    - type: attack
      hit:
        - type: damage
          damage: ...
```
the inner damage should still crit if the inner attack crits, but this approach would prevent that from happening.